### DISCUSSION TITLE
Can I add an interaction when a link between nodes is clicked?

### DISCUSSION THREAD

**Discussion by yalgee** [mermaid-js/mermaid#5296] [Q&A]
It is possible to bind a click event to the nodes. but I don't see any for the links between nodes.
Is there any way I can add some actions when links between nodes are clicked?

**Comment by yalgee** [comment]
I have found that I can and <a> tag to the link text. But not directly on the link(or arrow)

**Comment by Piratepie** [comment]
Having looked through the documentation and the implementation of the flowchart I can't see a way to add links or html elements to the links between nodes. I would suggest you make a feature request, since it does sound like a good idea.

**Comment by liaodalin19903** [comment]
> It is possible to bind a click event to the nodes. but I don't see any for the links between nodes. Is there any way I can add some actions when links between nodes are clicked?

@yalgee  Hi, how to bind click event to the nodes?
